test/run-integration.js

```diff
@@ -104,7 +104,7 @@ if (!isABrowser) {
 
 function initPlaces(placesAppId, placesApiKey) {
   return function(t) {
-    t.plan(1);
+    t.plan(2);
```

Comment:
I am unclear on what `t.plan` do. Can anyone explain the purpose of that method for me ? Thanks!

Reply:
it is to tell the test runner how many assertions will be done in this test, so that it knows when to bail in asynchronous contexts